docs/source/docs/explanation/formats/segment_anything.md

```diff
@@ -0,0 +1,117 @@
+# Segment Anything
+
+## Format specification
+
+SA-1B format specification is available [here](https://github.com/facebookresearch/segment-anything#dataset)
+
+The SA-1B is an dataset consisting of 1 billion segmentation masks model generated.
+The model is known as [SAM](https://ai.facebook.com/research/publications/segment-anything/) from FAIR and
+it can produce reliable and accurate segmentation masks given user prompt,
+such as point, box, natual sentence.
+
+Supported media types:
+
```

Comment:
please remove the empty line.

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1009/commits/652cd5d8027f8cef4c1545191bcc47dcae9278d0

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1009/commits/652cd5d8027f8cef4c1545191bcc47dcae9278d0